src/design-system/styles/template/block-areas/block-areas-annotate.scss

```diff
@@ -0,0 +1,42 @@
+@import "example-init";
+
+[data-block] {
+  position: relative;
+  outline: 2px dashed govuk-colour("grey-1");
+  padding: govuk-spacing(3);
+  padding-top: 3em;
```

Comment:
Could all `em` be set in `px` so that we're not mixing pixels (from `govuk-spacing`) and ems.

Reply:
Since this padding is relative to the size of the text, I think it's sensible here.